packages/coil-extension/src/background/services/AuthService.ts

```diff
@@ -153,17 +157,12 @@ export class AuthService extends EventEmitter {
       )
       this.trace('after refreshTokenAndUpdateWhoAmi', token)
     } else {
-      // Routinely do a whoami query to check for subscription status
+      // Routinely do a whoami query to check for subscription status or
```

Comment:
The Coil backend doesn't have GraphQL subscriptions set up for field changes?

Reply:
no, we don't have graphQL subscriptions set up